packages/storage/amplify_storage_s3/example/integration_test/transfer_acceleration.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:amplify_core/amplify_core.dart';
+import 'package:amplify_storage_s3_dart/amplify_storage_s3_dart.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:http/http.dart' as http;
+import 'package:tuple/tuple.dart';
+
+void testTransferAcceleration({
+  List<Tuple4<String, StorageAccessLevel, S3DataPayload, List<int>>>
+      dataPayloads = const [],
+  List<Tuple4<String, StorageAccessLevel, AWSFile, List<int>>> awsFiles =
+      const [],
+}) {
+  if (dataPayloads.isNotEmpty) {
```

Comment:
Why would these be empty? Can probably just remove this

Reply:
Ah yea wanted to make it generic, but I guess it's unnecessary for this test cases.